platform/pulp/app/serializers/repository.py

```diff
@@ -112,9 +112,9 @@ class ImporterSerializer(MasterModelSerializer):
         write_only=True,
         required=False,
     )
-    download_policy = serializers.MultipleChoiceField(
+    download_policy = serializers.ChoiceField(
```

Comment:
I must have git stashed this fix out of existence. This is the problem that caused full_clean to raise an exception early in the validation discussion.